### PR TITLE
add note about missing step

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1729

### CODE REVIEW

**Comment by e0i** [comment]
@VLG17 Please change the target branch of the PR from `live` to `master` to continue with the review.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/international/Set-WinUILanguageOverride.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ PS C:\> Set-WinUILanguageOverride
 
 This command sets the Windows UI language override to null for the current user account.
 
+> [!NOTE]
+> Logging off and back on is required for changes to take place.
```
Comment (dariomws) [inline docset/windows/international/Set-WinUILanguageOverride.md]:
```suggestion
```

**Comment by get-itips** [comment]
I am not seeing the suggested changes in the last commit... only a new line... I am confused

**Pull request merged**